docs/troubleshooting/index.md

```diff
@@ -41,7 +41,7 @@ This endpoint can be used on both the db nodes as well as the coordinator/query
 To use this, simply run the following on either the M3DB debug listen port or the regular port on M3Coordinator.
```

Comment:
Can you add something along the lines of "if the placement you are interested in is under a separate `env` than the service you are downloading the dump from, you may need to set the `Cluster-Environment-Name` header"

Reply:
Never mind! The fallback behavior isn't consistent.